### PR TITLE
extension name and description added

### PR DESCRIPTION
**Pull request by heimseiten**

### CODE REVIEW

File: linter/allowlists/default.txt [hunk 1]
```diff
@@ -52,6 +53,7 @@ Cross
 CSS
 CSV
 Dalley
+Debug
```
Comment (xchs) [inline linter/allowlists/default.txt:56]: This should go to the German or English allowlist.

File: linter/allowlists/default.txt [hunk 2]
```diff
@@ -242,6 +244,7 @@ sql
 start
 Start
 Stardate
+Sticky
```
Comment (xchs) [inline linter/allowlists/default.txt:247]: This should go to the German or English allowlist.

File: linter/allowlists/default.txt [hunk 3]
```diff
@@ -18,6 +18,7 @@ AuthCode
 AWC
 Back end
 back end
+Backend
```
Comment (xchs) [inline linter/allowlists/default.txt:21]: Why did you put this here? "Backend" is already available in the German allowlist: https://github.com/contao/package-metadata/blob/main/linter/allowlists/de.txt

In English, we used to use the notation "back end".

File: meta/heimseiten/contao-sticky-footer-backend-bundle/en.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+en:
+    title: Sticky Backend Footer
```
Comment (xchs) [inline meta/heimseiten/contao-sticky-footer-backend-bundle/en.yml:2]:
```suggestion
    title: Sticky Back End Footer
```

File: meta/heimseiten/contao-sticky-footer-backend-bundle/en.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+    description: >
+        Sticky footer for the backend with pure css. The footer is displayed immediately and not only after scrolling. Works also in debug mode.
```
Comment (xchs) [inline meta/heimseiten/contao-sticky-footer-backend-bundle/en.yml:4]:
```suggestion
        Sticky footer for the back end with pure CSS. The footer is displayed immediately and not only after scrolling. Works also in debug mode.
```

File: meta/heimseiten/contao-sticky-footer-backend-bundle/en.yml [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+    keywords:
+        - Backend
```
Comment (xchs) [inline meta/heimseiten/contao-sticky-footer-backend-bundle/en.yml:6]:
```suggestion
        - Back end
```

File: linter/allowlists/default.txt (hunk 1 above)
Comment (heimseiten) [reply linter/allowlists/default.txt:56]: Thank you!

File: linter/allowlists/default.txt (hunk 2 above)
Comment (heimseiten) [reply linter/allowlists/default.txt:247]: "Sticky" is in the name of the extension, which is the same in german and english. So it belongs to the default.txt?

File: linter/allowlists/default.txt (hunk 3 above)
Comment (heimseiten) [reply linter/allowlists/default.txt:21]: Thank you!

File: linter/allowlists/default.txt (hunk 2 above)
Comment (xchs) [reply linter/allowlists/default.txt:246]: "Sticky", is an English adjective. But, well... :)

**Review by xchs — APPROVED** [review]

**Pull request merged**